server/src/test/java/kabopok/server/HttpSteps.java

```diff
@@ -0,0 +1,65 @@
+package kabopok.server;
+
+import generated.kabopok.server.api.model.LoginDataDTO;
+import generated.kabopok.server.api.model.UserDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.io.Resource;
+import org.springframework.http.*;
```

Comment:
Import * Bad practic, please set in Android studio settings

Reply:
done